run/websockets/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC. All rights reserved.
+# Use of this source code is governed by the Apache 2.0
+# license that can be found in the LICENSE file.
+
+# Use the official lightweight Node.js 10 image.
+# https://hub.docker.com/_/node
+FROM node:12-slim
```

Comment:
Most samples are standardized on `node:12-slim`, helloworld is using `node:14-slim`, and 14 has been the LTS release for a while now.

suggestion: use node:14-slim and sync Dockerfile comment to base image tag.

```suggestion
# Use the official lightweight Node.js 14 image.
# https://hub.docker.com/_/node
FROM node:14-slim
```

Reply:
Deleted the dockerfile